LICENSE

```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
```

Comment:
I think we're free to change this. This is just the default template.

Reply:
👍 I'll let this to @etejedor as he his section looked into this for the spark monitor

Reply:
I'd use the same license as the one of the mono repo where this will probably end up.

https://github.com/swan-cern/jupyter-extensions/blob/master/LICENSE

The current Spark connector (which this is meant to replace some day) has that one.